kernel/kernel-api/src/test/scala/io/delta/kernel/internal/util/SchemaUtilsSuite.scala

```diff
@@ -55,6 +55,9 @@ class SchemaUtilsSuite extends AnyFunSuite {
       s"Error message '$msg' didn't contain: $shouldContain")
   }
 
+  private def validateSchemaWithDefaultSetting(schema: StructType): Unit =
```

Comment:
this can just be `validateSchema(schema:StructType, firstBooleanParam:Boolean = false, ...)